fedimint-client/Cargo.toml

```diff
@@ -42,6 +42,7 @@ thiserror = { workspace = true }
 tokio = { version = "1.36.0", features = [ "time", "macros" ] }
 tokio-stream = { version = "0.1.14", features = [ "time", "sync" ] }
 tracing = { workspace = true }
+reqwest = { version = "0.12.2", features = ["json", "rustls-tls"] }
```

Comment:
You'll have to deactivate default features here so that OpenSSL isn't included.

Reply:
oh right, I have become so lazy with `cargo add` now